HelpMyStreetFE/HelpMyStreetFE/Views/Shared/Components/JobDetailOutput/Default.cshtml

```diff
@@ -0,0 +1,94 @@
+﻿@model HelpMyStreetFE.Models.Account.Jobs.JobDetailViewModel
+@using HelpMyStreetFE.Helpers;
+@using HelpMyStreet.Utils.Enums;
+@using HelpMyStreet.Utils.Extensions;
+@using HelpMyStreet.Utils.Utils;
+@using Westwind.AspNetCore.Markdown
+
+@{
+    Layout = "_LayoutPrint";
+    var questionsToDisplay = Model.JobDetail.JobSummary.Questions.Where(q => q.ShowOnTaskManagement(false) && !string.IsNullOrEmpty(q.Answer));
+    var shoppingList = questionsToDisplay.Where(x => x.Id == (int)Questions.Shopping_List).FirstOrDefault();
+    questionsToDisplay = questionsToDisplay.Where(x => x.Id != (int)Questions.Shopping_List);
+}
+
+@if (!String.IsNullOrEmpty(Model.JobDetail.JobSummary.RecipientOrganisation))
+{
+    <h4>Group Name: @Model.JobDetail.JobSummary.RecipientOrganisation </h4>
+}
+<h4>Activity: @Model.JobDetail.JobSummary.SupportActivity.FriendlyNameShort()</h4>
+
+<h4>Request Details:</h4>
+<dl>
+    <dt><u>When Is The Help Needed?</u></dt>
```

Comment:
This all looks rather like the existing JobDetailViewComponent.  Are we not able to re-use that?